src/main/steps/tasklistresponse/allegations-of-harm-and-violence/check-your-answers/helpers/satetyConcernHelper.ts

```diff
@@ -28,7 +30,10 @@ export const HTMLParser = (keys, FoundElement: ANYTYPE, bodyHtml, userCase, type
         Array.isArray(FoundElement['childrenConcernedAbout']) &&
         FoundElement['childrenConcernedAbout'][0] === 'All the children in application'
       ) {
-        bodyHtml += HTML.LIST_ITEM + FoundElement['childrenConcernedAbout'][0] + HTML.LIST_ITEM_END;
+        bodyHtml +=
+          HTML.LIST_ITEM +
+          (language === 'cy' ? 'Pob plentyn yn y cais' : FoundElement['childrenConcernedAbout'][0]) +
```

Comment:
Can u not hard code it. Pls use infrastructure to handle translations

Reply:
Changed to use translation infrastructure